drivers/console/aarch64/console.S

```diff
@@ -95,10 +95,15 @@ endfunc console_getc
 	 * data that hasn't been output. It returns 0
 	 * upon successful completion, otherwise it
 	 * returns -1.
-	 * Clobber list : x0, x1
+	 * Clobber list : x0, x1, x2, x3, x4
 	 * ---------------------------------------------
 	 */
 func console_flush
+#if COREBOOT
+	mov	x4, x30
+	bl	coreboot_cbmc_flush
+	mov	x30, x4
+#endif
```

Comment:
I would expect all console drivers that need to support splitting would move the body of their `console_core_putc()` (and other functions) into exported functions with unique names in a new file (e.g. `ti16550_console_core_putc()` in `16550_multi_console.S`). The splitting driver would provide a strong definition of `console_core_putc()`, which would iterate through all the registered splitting drivers. The platform ports would include the new file for each console driver instead of the existing one.

Thinking about it some more, perhaps this could be an extension to the existing API using the name "multi console API" instead of "splitting". In hindsight it would be good if the existing API to supported multiple consoles like this but changing the existing APIs would not be backwards compatible for platforms.